### ISSUE TITLE
Global update on excluded categories 

### ISSUE THREAD

**Issue by LuR3n** [mihonapp/mihon#109]
### Steps to reproduce

Global update (with smart update turned off due to malfunction) updates my backlog category, which has been excluded in the settings.

### Expected behavior

Only the included categories should be updated

### Actual behavior

Every category gets updated

### Mihon version

0.16.1

### Android version

Android 13 

### Device

Samsung Galaxy A33 5G

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an official extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/extensions/issues/new/choose).
- [X] I have gone through the [FAQ](https://mihon.app/docs/faq/general) and [troubleshooting guide](https://mihon.app/docs/guides/troubleshooting/).
- [X] I have updated the app to version **[0.16.1](https://github.com/mihonapp/mihon/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I will fill out all of the requested information in this form.

**Comment by MajorTanya** [comment]
Are you talking about the Global Update or the scheduled automatic one?

**Comment by LuR3n** [comment]
> Are you talking about the Global Update or the scheduled automatic one?

Yeah, scheduled automatic update, my bad

**Comment by MajorTanya** [comment]
> Yeah, scheduled automatic update, my bad

I see. In your update settings, you only have the categories you want under `Included` right?

**Comment by LuR3n** [comment]
Yeah 
![Screenshot_20240117_213819_Mihon](https://github.com/mihonapp/mihon/assets/68476367/950c859d-4b1d-4486-80c3-79267359433e)
Should look like this, right? At least that's how it worked on tachi

**Comment by MajorTanya** [comment]
Yeah it should look something like that, could you maybe remove the excluded ones (make Excluded empty and Included only have Reading and Ongoing) and see what happens next time the scheduled run comes around?
Shouldn't be an issue but if that is, we need to figure it out

**Comment by LuR3n** [comment]
Alright, I'll try that and let you know
